js/reports-script.js

```
@@ -2,6 +2,71 @@
+    //     return;
+    // }
+
+    console.log("loading publish_reports_stats");
+
+    $.getJSON('/api/index.php?get=publish_reports_stats')
+        .done(await function (json) {
+            if (json && json.results) {
+                populateFilterOptions(json.results);
+            }
+        })
+        .fail(function (xhr, status, error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)